packages/core/src/filters/FilterSystem.ts

```diff
@@ -337,10 +351,17 @@ export class FilterSystem implements ISystem
 
             filters[i].apply(this, flip, lastState.renderTexture, CLEAR_MODES.BLEND, state);
 
+            if (i > 1 && state.multisample > 1)
```

Comment:
I usually store this thing as a single flag, so its guaranteed that we actually were in previous if. I understand its obvious in this case, but we also have things like `(1*2)+1` in our eslint, so maybe we can be careful here too?